### ISSUE TITLE
consider upating net-ldap to ~> 0.8.0 and rubyntlm to ~> 0.3.4

### ISSUE THREAD

**Issue by pravi** [intridea/omniauth-ldap#61]
We try to ship only one version  of a library in debian and we already updated net-ldap to 0.8.0 and 
rubyntlm to 0.3.4

**Comment by grosser** [comment]
@pyu10055 can you update this or would you merge a PR to update it ?

**Comment by grosser** [comment]
```
Name: net-ldap
Version: 0.3.1
Advisory: OSVDB-106108
Criticality: Low
URL: http://osvdb.org/show/osvdb/106108
Title: Net::LDAP for Ruby lib/net/ldap/password.rb SSHA Password Generation Weak Salt
Solution: upgrade to >= 0.6.0
```

**Comment by stakach** [comment]
+1 for this

**Comment by grosser** [comment]
@balasankarc can you merge this please ?

**Comment by grosser** [comment]
@pyu10055 no longer has access to the repo

**Comment by balasankarc** [comment]
@grosser I also don't have access on this repo (I don't see a reason for me to have access)

**Comment by balasankarc** [comment]
If I had access I wouldn't have created a PR !64

**Comment by grosser** [comment]
damn ... I figured since you are listed under members you had access :D

**Comment by pravi** [comment]
@grosser if this repo is officially unmaintained, everyone can switch to https://rubygems.org/gems/gitlab_omniauth-ldap which is well maintained. It would be a good idea to ask the maintainers of this project to give access to gitlab folks, so they can maintain this repo.

**Comment by grosser** [comment]
That sounds like a good alternative, I reached out to their support, let's
see if someone responds :)

**Comment by pravi** [comment]
@pyu1005 can you give access to rubygems.org omniauth-ldap project to gitlab_omniauth-ldap maintainers, so we can use gitlab_omniauth-ldap repo for pushing new versions. If you agree they could rename it to omniauth-ldap.

**Comment by pravi** [comment]
@grosser I work closely with gitlab folks for creating a native debian package of gitlab and they are very much responsive and helpful. In debian, we already include their patches.

**Comment by miketierney** [comment]
@grosser @pravi I've merged in the PR's and have a patch ready to go out. Unfortunately I don't have rubygems.org access (yet), but I hope to have that resolved later today.

Sorry for the delayed response.

**Comment by grosser** [comment]
:heart: thx for merging and taking the time to resolve this :)

**Comment by miketierney** [comment]
Released v1.0.5.

**Comment by grosser** [comment]
:heart: it finally happened :D
